src/ast/value/datetime.rs

```diff
@@ -280,6 +280,7 @@ pub struct ParsedTimestamp {
     pub minute: u8,
     pub second: u8,
     pub nano: u32,
+    pub timezone_offset_second: i64,
```

Comment:
Looking at the code that you've got and based on our earlier conversation, I think that it would be straightforward, and satisfy the desire to be information-preserving, to turn this into something like:

```
pub strcut ParseDateTime {
    -- 8< --
    timezone: Timezone,
}

enum Timezone {
    // I suppose that the none case is deterministic from
    // whether this is a TimezoneTz or a TimeZone, so this might not matter?
    None,
    OffsetSeconds(i64),
    Name(String),
}
```

We are certainly _eventually_ going to need to return this information, so I'd rather get it right now than force an api change down the road.

Reply:
we talked offline and decided not to go this route for now